Classes/CRON/FormBuilder/Controller/FormBuilderController.php

```diff
@@ -130,6 +134,37 @@ protected function createMailData($node, $data) {
 	}
 
 
+
+	/**
+	 * Checks the HMAC for the form submitted data
+	 */
+	protected function checkHmac() {
+
+		$trustedProperties = $this->request->getInternalArgument('__trustedProperties');
+
+		if ($trustedProperties) {
+
+			if ($this->request->hasArgument('data')
```

Comment:
Muss man das selbst machen? Gibt es da nicht eine API im Flow die das checken kann? Vor allen erzeugst du __trustedProperties ja nicht selbst, auch die Methode wie der HMAC ausgerechnet wird ist nicht deine. Daher wenn sich das intern in Flow ändert, müsste man das hier nachziehen.